modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+
+With this update, Real-time Collaboration now also checks for straight-up connectivity. And, if the connection goes down for 30 seconds, the ‘connection lost’ alert appears.
+
+For information on the **Real-Time Collaboration** plugin, see: xref:real-timecollaboration.adoc[Real-Time Collaboration plugin].
+
+
+=== Spell Checker Pro 3.0.3
+
+The {productname} 6.1 release includes an accompanying release of the **Spell Checker Pro** premium plugin.
+
+**Spell Checker Pro** 3.0.3 includes the following bug fix:
+
+==== An issue with the `DomDescent.toLeaf` API could result in Spell Checker Pro’s ‘preserve selection’ logic setting the selection when it did not need to
+
+The `DomDescent.toLeaf` API can walk inside void elements (for example `+<br>+`). The Core editor’s selection logic, by contrast, does not allow this.
+
+Consequently, when comparing selections where one is from `editor.selection.getRng()` and the other is created by `DomDescent.toLeaf`, the comparison can return `false` when it should be `true`.
+
+When this happens, code runs when it does not need to: in this case, Spell Checker Pro’s ‘preserve selection’ logic.
+
+With this update, SpellcheckerPro now correctly handles `DomDescent.toLeaf` being able to walk into a void element, making the range comparison more accurate, and ensuring the preserve selection code does not run when not needed.
+
+For information on the **Spell Checker Pro** plugin, see: xref:spellcheckerpro.adoc[Spell Checker Pro plugin].
+
+
+=== Table of Contents 1.1.0
+
+The {productname} 6.1 release includes an accompanying release of the **Table of Contents** premium plugin.
+
+**Table of Contents** 1.1.0 includes the following change:
+
+As part of making it possible to link Comments to almost any block element in a {productname} document, a `data-mce-toc` attribute is now added to the Table of Contents wrapper `div` while editing.
+
+This is an internal attribute, used as part of keeping comments attributed to the correct block element. It has no user- or integrator-facing impact.
+
+For information on the **Table of Contents** plugin, see: xref:tableofcontents.adoc[Table of Contents plugin].
+
+
+
+
+
+[[accompanying-premium-skins-and-icon-packs-changes]]
+== Accompanying Premium Skins and Icon Packs changes
+
+The {productname} 6.1 release includes an accompanying release of the **Premium Skins and Icon Packs**.
+
+=== Premium Skins and Icon Packs
+
+The **Premium Skins and Icon Packs** release includes the following bug fix:
+
+When using a Premium skin and the xref:comments.adoc[Comments plugin], avatars in the comments overflowed their bounds and covered the comment text in the Comments side-bar.
+
+This **Premium Skins and Icons Packs** release has been re-built to pull in the changes required to correct this.
+
+When a Premium skin and the Comments plugin are now used together, avatars stay within their bounds as they do in the {productname} default skin, and comment text is not hidden.
```

Comment:
This wasn't a fix done in this release, so this shouldn't be mentioned here. This was only a temporarily dev issue while working on 6.1 changes.

For the changes in the release, it's just a rebuild to pull in the `oxide` changes done for 6.1 as a whole.

Reply:
Edited to remove the ‘found and presented internally only’ bug documentation and made the ‘rolled the Oxide changes into the Premium Packs’ material explicit.